From 1a0b793b3f5a72bdcac08d841b2d1fdd0124e75b Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Thu, 22 Feb 2018 22:28:03 +0000
Subject: [PATCH] TRITON-174 Move NAPI to new Triton origin image Reviewed by:
 Julien Gilli <julien.gilli@joyent.com> Approved by: Julien Gilli
 <julien.gilli@joyent.com>

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index b878482..d4c6df5 100644
--- a/Makefile
+++ b/Makefile
@@ -40,9 +40,9 @@ BASH_FILES	:= sbin/napid bin/napictl
 JSON_FILES  := package.json config.json.sample
 
 ifeq ($(shell uname -s),SunOS)
-	# Allow building on a SmartOS image other than sdc-smartos/1.6.3.
-	NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
-	NODE_PREBUILT_VERSION=v0.10.32
+	# Allow building on a SmartOS image other than sdc-*-multiarch 15.4.1.
+	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
+	NODE_PREBUILT_VERSION=v0.10.48
 	NODE_PREBUILT_TAG := zone
 endif
 
-- 
2.21.0

